From 2a3ccd80e38a731c4fa5949318f7ca6d826c9c6c Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Thu, 4 Aug 2016 16:04:35 +0100
Subject: [PATCH] util: Stop zeroing the .a files, this breaks golang badly

The .a files are being modified post-install, whereas we should be relying
upon the toolchain itself to have deterministic builds, not as a hack later
on.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 pisi/util.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/pisi/util.py b/pisi/util.py
index d4296a8..d0bbb06 100644
--- a/pisi/util.py
+++ b/pisi/util.py
@@ -394,10 +394,6 @@ def calculate_hash(path):
         ctx.ui.info(_("Including directory '%s'") % path)
         value = None
     else:
-        if path.endswith('.a'):
-            # .a file content changes with each compile due to timestamps
-            # We pad them with zeroes, thus hash will be stable
-            clean_ar_timestamps(path)
         value = sha1_file(path)
 
     return (path, value)
-- 
2.9.2

